pkg/grafana/dashboards.go

```diff
@@ -52,6 +52,49 @@ func getRemoteDashboard(uid string) (*grizzly.Resource, error) {
 	return &resource, nil
 }
 
+func getRemoteDashboardList() ([]string, error) {
+	batchSize := 500
+
+	UIDs := []string{}
+	for page := 1; ; page++ {
+		grafanaURL, err := getGrafanaURL(fmt.Sprintf("/api/search?type=dash-db&limit=%d&page=%d", batchSize, page))
+		if err != nil {
+			return nil, err
+		}
+
+		resp, err := http.Get(grafanaURL)
+		if err != nil {
+			return nil, err
+		}
+		defer resp.Body.Close()
+
+		switch resp.StatusCode {
+		case http.StatusNotFound:
+			return nil, grizzly.ErrNotFound
+		default:
+			if resp.StatusCode >= 400 {
+				return nil, errors.New(resp.Status)
+			}
+		}
```

Comment:
```suggestion
		switch resp.StatusCode {
		case http.StatusNotFound:
			return nil, grizzly.ErrNotFound
		case resp.StatusCode >= 400:
			return nil, errors.New(resp.Status)
		}
```

Reply:
really? My IDE tells me that's not valid (`cannot convert resp.StatusCode >= 400 (untyped bool value) to int`)

Reply:
oh my bad, sorry for that. The following should do:

```go
switch {
    case resp.StatusCode == http.StatusNotFound:
      // sth
    case resp.StatusCode >= 400:
      // sth
}